e2e/utils.go

```diff
@@ -508,6 +508,60 @@ func validatePVCAndAppBinding(pvcPath, appPath string, f *framework.Framework) e
 	return err
 }
 
+func validatePVCAndAppWaitForFirstConsumer(pvcPath, appPath string, f *framework.Framework) error {
+	pvc, err := loadPVC(pvcPath)
+	if err != nil {
+		return err
+	}
+	pvc.Namespace = f.UniqueName
+
+	app, err := loadApp(appPath)
+	if err != nil {
+		return err
+	}
+	app.Namespace = f.UniqueName
+
+	err = createPVC(f.ClientSet, pvc)
+	if err != nil {
+		return err
+	}
+
+	//verifies that the pvc is in pending state
```

Comment:
Comments are mostly written with a space after `//`.

This comment here does not add much value, the function is pretty much self explaining. It would be better to add a comment above the function itself, which explains that `validatePVCAndAppWaitForFirstConsumer` 1st waits for pending, then starts an app and then makes sure the claim is bound.